### PR TITLE
OKTA-438749 multi to single custom widget

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Converting the "style yer widget" pages from many to one.
- **Is this PR related to a Monolith release?** Nein.

### Resolves:

* [OKTA-438749](https://oktainc.atlassian.net/browse/OKTA-438749)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/protect-your-api/next-steps/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ You should now understand how to protect your API.
 
 Read about customization options:
 
-* [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/)
+* [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/protect-your-api/next-steps/index.md:8]:
```suggestion
* [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/main/index.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ You'll also need the full redirect URI that you defined in the [Define a callbac
 
 To sign in users, open a browser and navigate to the Okta-hosted sign-in page.
 
-To learn how to customize the sign-in page, see [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/).
+To learn how to customize the sign-in page, see [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/main/index.md:91]:
```suggestion
To learn how to customize the sign-in page, see [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/main/index.md [hunk 3]
```diff
@@ -132,5 +132,5 @@ Your API must check for valid tokens in incoming requests. To learn how to prote
 
 You should now understand how to sign users in to your mobile applications using Okta.
 
-* In this guide you signed users in to your app by opening a browser. To learn how to customize the sign-in page displayed in the browser, see [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/).
+* In this guide you signed users in to your app by opening a browser. To learn how to customize the sign-in page displayed in the browser, see [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/main/index.md:135]:
```suggestion
* In this guide you signed users in to your app by opening a browser. To learn how to customize the sign-in page displayed in the browser, see [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/index.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ You need two values from the Okta application and the Admin Console that you wor
 
 In your application, you want to provide a button to sign the user in. When the user clicks this button, they are redirected to the Okta-hosted sign-in page so they can authenticate.
 
-> **Note:** To customize this sign-in page, see the [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/).
+> **Note:** To customize this sign-in page, see the [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/index.md:115]:
```suggestion
> **Note:** To customize this sign-in page, see the [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/index.md [hunk 5]
```diff
@@ -180,5 +180,5 @@ You should now understand how to sign users in to your single-page applications
 From here, go on to check out these guides:
 
 * [Customize the Okta URL domain](/docs/guides/custom-url-domain/)
-* [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/)
+* [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/main/index.md:183]:
```suggestion
* [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md [hunk 6]
```diff
@@ -134,7 +134,7 @@ To sign a user in, your application must redirect the browser to the Okta-hosted
 
 The user is redirected to the hosted sign-in page where they authenticate. After successful authentication, the browser is redirected back to your application along with information about the user.
 
-> **Note:** To customize the hosted sign-in page, see [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/).
+> **Note:** To customize the hosted sign-in page, see [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md:137]:
```suggestion
> **Note:** To customize the hosted sign-in page, see [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md [hunk 7]
```diff
@@ -171,5 +171,5 @@ You should now understand how to sign users in to your web applications using Ok
 Read more:
 
 * [Customize the Okta URL domain](/docs/guides/custom-url-domain/)
-* [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/)
+* [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md:174]:
```suggestion
* [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md [hunk 8]
```diff
@@ -78,4 +78,4 @@ You might want to configure self-service registration for your org:
 Read more on customizing your org:
 
 * [Customize the Okta URL domain](/docs/guides/custom-url-domain/)
-* [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/)
\ No newline at end of file
+* [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md:81]:
```suggestion
* [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 9]
```diff
@@ -3012,7 +3012,7 @@ curl -v -X POST \
 
 #### Enroll WebAuthn Factor
 
-> **Note:** The WebAuthN Factor is available for those using the [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/). If you are using a self-hosted, customized sign-in widget, you must first upgrade to widget version 3.4.0 and enable the [configuration option](https://github.com/okta/okta-signin-widget#configuration).
+> **Note:** The WebAuthN Factor is available for those using the [Style the Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget). If you are using a self-hosted, customized sign-in widget, you must first upgrade to widget version 3.4.0 and enable the [configuration option](https://github.com/okta/okta-signin-widget#configuration).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:3015]:
```suggestion
> **Note:** The WebAuthN Factor is available for those using the [Style the Okta-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget). If you are using a self-hosted, customized sign-in widget, you must first upgrade to widget version 3.4.0 and enable the [configuration option](https://github.com/okta/okta-signin-widget#configuration).
```

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
What you have in the conductor.yml file looks good, but there are a few links **to** `*/style-the-widget/*` (line 2083 - 2089) that needs to be updated. A few nits on the headings, but other than that, the rest looks good!

**Comment by brentschaus-okta** [comment]
Test comment from mobile app.

**Comment by brentschaus-okta** [comment]
> What you have in the conductor.yml file looks good, but there are a few links **to** `*/style-the-widget/*` (line 2083 - 2089) that needs to be updated. A few nits on the headings, but other than that, the rest looks good!

Great catch. Thanks!

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-styling/main/index.md [hunk 10]
```diff
@@ -21,6 +21,6 @@ Find out how to modify and customize the look of the Okta Sign-In Widget.
...
+After you upgrade your Sign-In Widget to Okta Identity Engine, you can modify the look of the initial sign-in page. See [Style the self-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget) for examples and information on how to [modify the initial sign-in page](/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget#initial-sign-in-page) and how to make your own customizations to the Widget such as [increasing the Widget's width](/docs/guides/custom-widget/main/#widget-positioning-width) and [adding a background color](/docs/guides/custom-widget/main/#background).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-styling/main/index.md:24]:
```suggestion
After you upgrade your Sign-In Widget to Okta Identity Engine, you can modify the look of the initial sign-in page. See [Style the self-hosted Sign-In Widget](/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget) for examples and information on how to [modify the initial sign-in page](/docs/guides/custom-widget/main/#initial-sign-in-page) and how to make your own customizations to the Widget such as [increasing the Widget's width](/docs/guides/custom-widget/main/#widget-positioning-width) and [adding a background color](/docs/guides/custom-widget/main/#background).
```

**Review by vanngo-okta — APPROVED** [review]
Sorry, I missed one last anchor. 
The rest LGTM! 🚀

**Pull request merged**